packages/material-ui/src/MenuItem/menuItemClasses.ts

```diff
@@ -3,12 +3,16 @@ import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unsty
 export interface MenuItemClasses {
   /** Styles applied to the root element. */
   root: string;
-  /** Styles applied to the root element unless `disableGutters={true}`. */
-  gutters: string;
-  /** Styles applied to the root element if `selected={true}`. */
-  selected: string;
-  /** Styles applied to the root element if dense. */
+  /** Pseudo-class applied to the `component`'s `focusVisibleClassName` prop. */
```

Comment:
I would propose we use the same description as the other components that are implemented in the fashion: 
```suggestion
  /** Pseudo-class applied to the ButtonBase root element if the button is keyboard focused. */
```